lsst/ts/cbp/component.py

```diff
@@ -409,6 +404,20 @@ def get_cbp_telemetry(self):
         self.get_mask()
         self.get_mask_rotation()
 
+    def configure(self):
```

Comment:
Consider having this method take `config` as an argument and save it as self.config. The former makes the control flow a bit easier to follow and the latter saves a lot of copying.

Note that you can avoid setting self.masks earlier on if you do this, as long as you insist that configure be called before the configuration is used.